README.md

```diff
@@ -32,3 +32,23 @@ Use of the `--dump-hashes` option is recommended as it avoids having to
 read the file system again if you need to re-run the command.  See
 `--read-hashes` option.  Also the dump file must be provided when users
 request help.
+
+#### Note
+
+The first `dd` command is meant to be used when trying to restore the
+filesystem on a different support or partition rather than the damaged one.
+This will copy the first portion of the damaged partition, i.e. the portion
+that was already succesfully moved, stopping right before the overlapping
+section would begin.
```

Comment:
This first paragraph is not correct.  The file system has been split
into 2 pieces by the interrupted GParted resize/move operation.
`find-overlap.py` provides the 2 `dd` commands to run to copy both
pieces into a new partition and put the file system back together.  Only
running the first `dd` command is loosing the 2 piece of the file
system.

Reply:
Answered [below](https://github.com/mfleetwo/find-overlap/pull/1#issuecomment-2522684361).